.github/workflows/quality.yml

```
@@ -0,0 +1,64 @@
+      uses: actions/checkout@v4
+      
+    - name: Run Gosec Security Scanner
+      uses: securego/gosec@master
+      with:
+        args: '-fmt sarif -out gosec-results.sarif ./...'
+        
+    - name: Upload SARIF file
+      uses: github/codeql-action/upload-sarif@v3
+      if: always()
+      with:
+        sarif_file: gosec-results.sarif
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)